Doc/whatsnew/3.11.rst

```diff
@@ -180,4 +185,4 @@ Removed
   parameter of functions :func:`~gettext.translation` and
   :func:`~gettext.install` are also removed, since they are only used for
   the ``l*gettext()`` functions.
-  (Contributed by Dong-hee Na and Serhiy Storchaka in :issue:`44235`.)
\ No newline at end of file
+  (Contributed by Dong-hee Na and Serhiy Storchaka in :issue:`44235`.)
```

Comment:
Why this change? :)

Reply:
Ask my text editor.

Reply:
My text editor (vim) likes to add newline at the end of a file.